dialect/entsql/annotation.go

```diff
@@ -86,6 +86,14 @@ type Annotation struct {
 	//	}
 	//
 	OnDelete ReferenceOption `json:"on_delete,omitempty"`
+
+	// Check allows injecting custom "DDL" for setting the "CHECK" clause in "CREATE TABLE"
+	// table that has matching rows in the child table.
+	//	entsql.Annotation{
+	//		Check: < 10,
```

Comment:
Is this line missing double-quotes?
```suggestion
	//		Check: "< 10",
```